lib/mongoid/lazy_migration/document.rb

```diff
@@ -63,8 +63,7 @@ def try_lock_migration
     selector = atomic_selector.merge('migration_state' => { "$in" => [nil, 'pending'] })
     changes  = { "$set" => { 'migration_state' => 'processing' }}
     safety   = { :safe => true }
-
-    self.class.with(safety).where(selector).query.update(changes)['updatedExisting']
+    self.class.with(safety).where(selector).update(changes).successful?
```

Comment:
https://docs.mongodb.org/v3.0/reference/command/getLastError/

`getLastError.updatedExisting`?

I don't see any reference to the `update` command anymore. I think there's `upsert` and `update_attributes`...